python/revng/restapi/README.md

```diff
@@ -0,0 +1,11 @@
+# rev.ng REST API
+
+## How to start a dev server
+
+To start a dev server:
+
+* Make sure that the `revng` module is in the `PYTHONPATH`
+* Run
+  ```shell
+  FLASK_APP=revng.restapi FLASK_ENV=development flask run
+  ```
```

Comment:
I think we want a `revng revngd` or `revng d` or `revng daemon` command. See `python/revng/cli/opt.py`.

Can `flask` get command line arguments? Maybe we can avoid the environment variable and let it fit in how we're doing things elsewhere (i.e., have the `revng` cli automatically add `-load` arguments for you).
This way, `revng.restapi` would not need to use `collect_files`, the *driver* would take care of it.

If we can't have arguments, we could still have `revng` CLI populate the environment variable so that `revng.restapi` is agnostic.

Then we can drop this file entirely.

Reply:
Dropped